sphere-models/src/it/java/io/sphere/sdk/products/ProductProjectionSearchFilterTypesIntegrationTest.java

```diff
@@ -346,11 +559,12 @@ public void filtersTimeSetAttributes() throws Exception {
     }
 
     @Test
-    public void filtersDateTimeSetAttributes() throws Exception {
+    public void dateTimeSetAttributesFilters() throws Exception {
         final ProductProjectionSearch termSearch = ProductProjectionSearch.ofStaged()
                 .withQueryFilters(model -> model.allVariants().attribute().ofDateTime(ATTR_NAME_DATE_TIME_SET)
                         .filtered().by(ZonedDateTime.parse("2002-10-12T23:06:10.204+00:00")));
```

Comment:
Product 1 has [2001-09-11T22:05:09.203+00:00, 2002-10-12T23:06:10.204+00:00]
Product 2 has [2001-09-11T22:05:09.203+00:00]
If we filter by 2002-10-12T23:06:10.204+00:00 only Product 1 should be returned.